survey_multiple_choice_max_answer/models/survey_question.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SurveyQuestion(models.Model):
+
+    _inherit = "survey.question"
+
+    validation_multiple_answers_min = fields.Integer(
+        "Minimum Number of Answers", default=0
+    )
+    validation_multiple_answers_max = fields.Integer("Maximum Number of Answers")
+
+    _sql_constraints = [
+        (
+            "non_neg_multiple_ans_min",
+            "CHECK (validation_multiple_answers_min >= 0)",
+            "The minimum number of answers must be non-negative!",
+        ),
+        (
+            "non_neg_multiple_ans_max",
+            "CHECK (validation_multiple_answers_max >= 0)",
+            "The maximum number of answers must be non-negative!",
+        ),
+        (
+            "validation_multiple_ans",
+            "CHECK (validation_multiple_answers_min <= validation_multiple_answers_max)",
+            "Max number of multiple answers cannot be smaller "
+            "than min number of multiple answers!",
+        ),
+    ]
+
+    def _validate_choice(self, answer, comment):
+        res = super()._validate_choice(answer, comment)
+        if res or self.question_type != "multiple_choice":
+            return res
+
+        # When only 1 suggested answer is received from website in controller,
+        # it should be given as a list of length 1: [sugg_ans_id],
+        # but it is given as a string: str(sugg_ans_id)
+        answer_count = len(answer) if isinstance(answer, list) else 1
+        if self.validation_required and not (
```

Comment:
I think you should check if validation_multiple_answers_max is different than 0. 
If your module and another that extends the validation functionality too are both installed, if the user doesn't set the values because he does not want to use them but instead leaves them at 0 (default value of an Integer) then the validation will fail.

Reply:
You're right. Changed + unit test added